reports/audit-runs/2026-08-28T0634Z-req-new2-61fa446/final-summary.md

```
@@ -0,0 +1,57 @@
+- REQ-TRACE-003 P1 drift invented (REQ-ARCH-030 anchor) - 12 docs + 9 src invented
+
+## Plan (3 waves, flat debt)
+Wave1: CSV sync for invented (or remove markers)
+Wave2: Tag existing tests for 26 concrete untraced
+Wave3: Harden top 20 generic orphan with concrete paths
+
+## Next
+Issues phase D requires approve to create up to 5 per iteration (MAX_ISSUES=5). This iteration produced 0 issues (plan owns).
+
+## Issues created (iter1)
+- #9803 [req][GAP][P1] drift (REQ-TRACE-003 GAP)
```

markdownlint-cli2 (0.23.2)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "p1"

(MD052, reference-links-images)